test/e2e/networking.go

```diff
@@ -18,25 +18,25 @@ package e2e
 
 import (
 	"fmt"
-	"net/http"
-	"strings"
-	"time"
-
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/fields"
 	"k8s.io/kubernetes/pkg/labels"
 	"k8s.io/kubernetes/pkg/util"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	"k8s.io/kubernetes/pkg/util/wait"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
 )
 
-var _ = Describe("Networking", func() {
-	f := NewFramework("nettest")
-
-	var svcname = "nettest"
+//versions ~ 1.3 (original RO test), 1.6 uses newer services/tokens,...
+const nettestVersion = "1.6"
 
+var _ = Describe("NetworkingNew", func() {
```

Comment:
Agree we can keep these as Networking/nettest....  No need to update name since we're splicing it back in.